test/test-clusters/src/main/java/org/elasticsearch/test/cluster/local/LocalClusterFactory.java

```diff
@@ -482,6 +486,28 @@ private void configureSecurity() {
                         "-r",
                         user.getRole()
                     );
+                    if (user.isOperator()) {
+                        operators.add(user.getUsername());
+                    }
+                }
+
+                if (operators.isEmpty() == false) {
+                    // TODO: Support service accounts here
+                    Path destination = workingDir.resolve("config").resolve("operator_users.yml");
+                    boolean fileExists = Files.exists(destination);
+                    try (Writer writer = Files.newBufferedWriter(destination, StandardOpenOption.CREATE, StandardOpenOption.APPEND)) {
+                        // If the file already exists (the test has its own operator_users file) then don't duplicate the top-level key
```

Comment:
This seems a bit brittle and sensitive to ordering. Should we make this behavior more explicit? Should we touch the file at all if a test has explicitly provided one?

Reply:
I think it would be a mistake to ignore the request to configure the users as operators.
It would lead to hard to diagnose failures.

I can make it an error to mark any users as operators if you also provide your own operators file.
That could be annoying in some cases, but at least it's obvious what's wrong.